controllers/configurationpolicy_controller.go

```diff
@@ -2410,16 +2410,16 @@ func handleSingleKey(
 	}
 
 	if key == "stringData" && existingObj.GetKind() == "Secret" {
-		// override automatic conversion from stringData to data prior to evaluation
-		existingValue = existingObj.UnstructuredContent()["data"]
-
+		// override automatic conversion from stringData to data before evaluation
 		encodedValue, _, err := unstructured.NestedStringMap(existingObj.Object, "data")
 		if err != nil {
 			message := "Error accessing encoded data"
 
 			return message, false, mergedValue, false
 		}
 
+		existingValue = make(map[string]interface{}, len(encodedValue))
+
```

Comment:
Should we change this https://github.com/open-cluster-management-io/config-policy-controller/blob/34c5a931b272a414aed605cafac95687e0c6b859/controllers/configurationpolicy_controller.go#L2432? because of this

Reply:
https://github.com/open-cluster-management-io/config-policy-controller/blob/34c5a931b272a414aed605cafac95687e0c6b859/controllers/configurationpolicy_controller.go#L2382C2-L2391C4 maybe affect here if `existingValue`  is still interface{}